### PR TITLE
Made link_id from cellmia consistent with other software

### LINKED ISSUE #50: cmso_link_id starts from 1 for CellMIA only
Just noticed this and checked examples of all tracking software folders.
The cmso_link_id in the links.csv all starts from 0 except in the [CellMIA example](https://github.com/CellMigStandOrg/biotracks/blob/master/examples/CELLMIA/example_1/dp/links.csv), where it starts from 1.

Probably requires just a tiny number change. I'll see if I can find the specific script file and line nr.

### PR DESCRIPTION
**Pull request by gsergeant**
Fixes #50, updated `links.csv` is included.

### CODE REVIEW

**Comment by simleo** [comment]
You also need to update the `cmso_link_id` column in `examples/CELLMIA/example_1/dp/tracks.csv`

**Comment by gsergeant** [comment]
Overlooked that because it does not automatically gets saved to disk when running `create_dpkg.` 
It's still the old file, will update, thanks.

File: biotracks/readfile.py [hunk 1]
```diff
@@ -385,6 +385,8 @@ def read(self):
         frame_id = self.conf[config.TRACKING].get(cmso.FRAME_ID)
         df = pd.read_csv(self.fname, sep=self.SEP, encoding=self.ENCODING,
                          usecols=[cellmia_link_id, frame_id, x, y])
+        # subtract 1 for consistency across software
+        df[cellmia_link_id] = df[cellmia_link_id].apply(lambda x: x-1)
```
Comment (simleo) [inline biotracks/readfile.py]: This can be shortened to `df[cellmia_link_id] -= 1`

**Review by simleo — CHANGES_REQUESTED** [review]

File: biotracks/readfile.py (hunk 1 above)
Comment (gsergeant) [reply biotracks/readfile.py]: Ok, wow I had not expected something that simple could work  🦆

**Comment by simleo** [comment]
```
% git show master:examples/CELLMIA/example_1/dp/links.csv >old_links.csv  
% python -c 'with open("examples/CELLMIA/example_1/dp/links.csv") as f: 
  lines = [_.strip().split(",") for _ in f]
with open("exp_old_links.csv", "w") as f: 
  f.write(",".join(lines.pop(0))+"\n")
  [f.write("%s,%d\n" % (x, int(y) + 1)) for (x, y) in lines]'
% cmp {,exp_}old_links.csv
```

Same as above for `tracks.csv`.

:+1:

**Pull request merged**